tests/e2e/tests/pilot/externalservice_test.go

```diff
@@ -58,37 +66,150 @@ func TestServiceEntry(t *testing.T) {
 			url:               "http://bing.com",
 			shouldBeReachable: false,
 		},
-		// See issue https://github.com/istio/istio/issues/7869
-		//{
-		//	name:              "REACHABLE_wikipedia.org_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://www.wikipedia.org",
-		//	shouldBeReachable: true,
-		//},
-		//{
-		//	name:              "UNREACHABLE_google.com_over_cidr_range",
-		//	config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
-		//	url:               "https://google.com",
-		//	shouldBeReachable: false,
-		//},
-		{
-			name:              "REACHABLE_en.wikipedia.org_over_wikipedia_wildcard",
-			config:            "testdata/networking/v1alpha3/wildcard-tls-wikipedia.yaml",
-			url:               "https://en.wikipedia.org/wiki/Main_Page",
+		{
+			name:              "UNREACHABLE_bing.com_over_bing_wildcard_443",
+			config:            "testdata/networking/v1alpha3/service-entry-wildcard-bing.yaml",
+			url:               "https://www.bing.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_wikipedia.org_over_cidr_range",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://www.wikipedia.org",
+			shouldBeReachable: true,
+		},
+		{
+			name:              "UNREACHABLE_google.com_over_cidr_range",
+			config:            "testdata/networking/v1alpha3/service-entry-tcp-wikipedia-cidr.yaml",
+			url:               "https://google.com",
+			shouldBeReachable: false,
+		},
+		{
+			name:              "REACHABLE_in.google.com_over_google_wildcard",
```

Comment:
sorry for the misdirection.. the tests are using google.co.in but we are actually testing for in.google.com (which doesn't exist)..
Lets switch this to yahoo.. As yahoo has proper sites for in.yahoo.com, uk.yahoo.com, us.yahoo.com, etc..
So we would have
REACHABLE_in.yahoo.com_443_over_yahoo_wildcard_443
UNREACHABLE_www.google.com_443_over_yahoo_wildcard_443

using service-entry-wildcard-yahoo.yaml
where the yaml file is like

```
hosts:
- *.yahoo.com
ports:
- number: 443
   protocol: TLS
   name: https
resolution: none
mesh_external
```

Reply:
This test above will check that https://in.yahoo.com can be reached with just the service entry without any virtual service.

Reply:
maybe I'm missing something, this site does exist:

```$ curl https://www.google.co.in
<!doctype html><html itemscope="" itemtype="http://schema.org/WebPage" lang="en"><head><meta content="Search the world's information, including webpages, images, videos and more. Google has many special features to help you find exactly what you're looking for." name="description"><meta content="noodp" name="robots"><meta content="text/html; charset=UTF-8" http-equiv="Content-Type"><meta content="/images/branding/googleg/1x/googleg_standard_color_128dp.png" itemprop="image"><title>Google</tit```

Reply:
and only exists when you have https://www.google.co.in or http://www.google.co.in.

If you do google.co.in, that will be a 304.